v1/firebase/adapter.js

```diff
@@ -40,7 +40,18 @@ function apiGetTrack(trackId) {
 
 function apiGetChannel(channelId) {
 	return apiGet(`channels/${channelId}`).then(snapshot => {
-		return serializeChannel(snapshot.val(), channelId);
+		var channel = serializeChannel(snapshot.val(), channelId);
+
+		// Replace `images` with a single `thumbnail` URL.
+		var images = Object.keys(channel.images);
+		var lastImage = images[images.length - 1];
```

Comment:
Yes, would be great to clean out old images. Since it'll probably take a while before we get that done, we can still merge this. And review again once new image-structure is in place. Let's open another issue for that.

I I were to build a player with this API I would want multiple image sizes. Should we do something like?

```
images: {
  small: '120px.jpg',
  medium: '320px.jpg',
  large: '1400px.jpg',
}
```